### PR TITLE
Added language TsundereGerman

### PR DESCRIPTION
**Pull request by MrMorkat**
translated the tsundere version of TillerinoBot into german
there are around 3-5 translations which i think are not perfect, i should have made a comment next to them

### CODE REVIEW

File: tillerinobot/src/main/java/tillerino/tillerinobot/lang/TsundereBase.java [hunk 1]
```diff
@@ -0,0 +1,281 @@
...
+import tillerino.tillerinobot.BeatmapMeta;
+import tillerino.tillerinobot.CommandHandler.Action;
+import tillerino.tillerinobot.CommandHandler.Message;
+import tillerino.tillerinobot.CommandHandler.NoResponse;
+import tillerino.tillerinobot.CommandHandler.Response;
+import tillerino.tillerinobot.RecommendationsManager.Recommendation;
+import tillerino.tillerinobot.diff.PercentageEstimates;
+
+public class TsundereBase implements Language
+{
```
Comment (omkelderman) [inline tillerinobot/src/main/java/tillerino/tillerinobot/lang/TsundereBase.java]: `{` at the end of the previous line please :smile:

File: tillerinobot/src/main/java/tillerino/tillerinobot/lang/TsundereBase.java [hunk 2]
```diff
@@ -0,0 +1,281 @@
...
+
+import tillerino.tillerinobot.BeatmapMeta;
+import tillerino.tillerinobot.CommandHandler.Action;
+import tillerino.tillerinobot.CommandHandler.Message;
+import tillerino.tillerinobot.CommandHandler.NoResponse;
+import tillerino.tillerinobot.CommandHandler.Response;
+import tillerino.tillerinobot.RecommendationsManager.Recommendation;
+import tillerino.tillerinobot.diff.PercentageEstimates;
+
+public class TsundereBase implements Language
```
Comment (omkelderman) [inline tillerinobot/src/main/java/tillerino/tillerinobot/lang/TsundereBase.java]: this should be an `abstract` class

File: tillerinobot/src/main/java/tillerino/tillerinobot/lang/TsundereBase.java (hunk 1 above)
Comment (MrMorkat) [inline tillerinobot/src/main/java/tillerino/tillerinobot/lang/TsundereBase.java]: haha, sure :')

File: tillerinobot/src/main/java/tillerino/tillerinobot/lang/Tsundere.java [hunk 3]
```diff
@@ -14,7 +14,7 @@
 import tillerino.tillerinobot.RecommendationsManager.Recommendation;
 import tillerino.tillerinobot.diff.PercentageEstimates;
 
-public class Tsundere implements Language {
+public class TsundereEnglish extends TsundereBase implements Language {
```
Comment (omkelderman) [inline tillerinobot/src/main/java/tillerino/tillerinobot/lang/Tsundere.java]: no need to implement the `Language` interface, `TsundereBase` already does

File: tillerinobot/src/main/java/tillerino/tillerinobot/lang/TsundereGerman.java [hunk 4]
```diff
@@ -0,0 +1,392 @@
...
+
+import tillerino.tillerinobot.BeatmapMeta;
+import tillerino.tillerinobot.CommandHandler.Action;
+import tillerino.tillerinobot.CommandHandler.Message;
+import tillerino.tillerinobot.CommandHandler.NoResponse;
+import tillerino.tillerinobot.CommandHandler.Response;
+import tillerino.tillerinobot.RecommendationsManager.Recommendation;
+import tillerino.tillerinobot.diff.PercentageEstimates;
+
+public class TsundereGerman extends TsundereBase implements Language {
```
Comment (omkelderman) [inline tillerinobot/src/main/java/tillerino/tillerinobot/lang/TsundereGerman.java]: no need to implement the `Language` interface, `TsundereBase` already does

**Pull request closed without merging**